caffe2/operators/utility_ops.h

```diff
@@ -738,6 +738,105 @@ class ScatterAssignOp : public Operator<Context> {
   INPUT_TAGS(DATA, INDICES, SLICES);
 };
 
+template <class Context>
+class ScatterOp : public Operator<Context> {
+ public:
+  USE_OPERATOR_CONTEXT_FUNCTIONS;
+
+  template <class... Args>
+  explicit ScatterOp(Args&&... args)
+      : Operator<Context>(std::forward<Args>(args)...),
+        OP_SINGLE_ARG(int, "axis", axis_, 1) {
+  }
+
+  virtual ~ScatterOp() noexcept {}
```

Comment:
In file included from caffe2/caffe2/operators/utility_ops.cc:1:
caffe2/caffe2/operators/utility_ops.h:752:11: warning: '~ScatterOp' overrides a destructor but is not marked 'override'

```
virtual ~ScatterOp() noexcept {}
        ^
caffe2/c10/util/Type.h:19:56: note: in instantiation of template class 'caffe2::ScatterOp<caffe2::CPUContext>' requested here
  static const auto& name = *(new std::string(demangle(typeid(T).name())));
                                                       ^
caffe2/caffe2/operators/utility_ops.cc:54:1: note: in instantiation of function template specialization 'c10::demangle_type<caffe2::ScatterOp<caffe2::CPUContext> >' requested here
REGISTER_CPU_OPERATOR(Scatter, ScatterOp<CPUContext>);
^
caffe2/caffe2/core/operator.h:1257:3: note: expanded from macro 'REGISTER_CPU_OPERATOR'
  C10_REGISTER_CLASS(CPUOperatorRegistry, name, __VA_ARGS__)
  ^
caffe2/c10/util/Registry.h:291:3: note: expanded from macro 'C10_REGISTER_CLASS'
  C10_REGISTER_TYPED_CLASS(RegistryName, #key, __VA_ARGS__)
  ^
caffe2/c10/util/Registry.h:251:14: note: expanded from macro 'C10_REGISTER_TYPED_CLASS'
      ::c10::demangle_type<__VA_ARGS__>());
             ^
caffe2/caffe2/core/operator.h:751:3: note: overridden virtual function is here
  ~Operator() noexcept override {}
  ^

```

Reply:
please address this comment as well, thanks.